src/main/java/org/trips/service_framework/dtos/SkuAttributes.java

```diff
@@ -0,0 +1,51 @@
+package org.trips.service_framework.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class SkuAttributes {
+    private String spec;
+    private String species;
+    private String productType;
+    private String freezingMethod;
+    private String packing;
+    private String glazingPercentage;
+    private String unitWeight;
+    private String quality;
+    private String treatment;
+    private String quantityPerUnit;
```

Comment:
Add a validation here that ensures we either get

- `quantityPerUnit` and `unitPerCarton`, or
- `unitWeight`

Any other combination of these fields is invalid and should not be processed.